child_compassion/__manifest__.py

```diff
@@ -60,7 +60,8 @@
         'views/gmc_message_view.xml',
         'views/compassion_reservation_view.xml',
         'views/field_office_disaster_alert_view.xml',
-        'views/res_config_view.xml',
+        'views/availability_management_view.xml',
```

Comment:
You should put this after compassion_settings_view, because it depends on it and should therefore be loaded after.